README.md

```diff
@@ -40,3 +40,21 @@ Contributions are very welcome, as are feature requests and suggestions. Please
 [codecov-url]: https://codecov.io/gh/JuliaData/CSV.jl
 
 [issues-url]: https://github.com/JuliaData/CSV.jl/issues
+
+## Alternatives
+
+There are several other packages for reading CSV files in Julia, which may suit your needs better:
+
+* The standard library contains [DelimitedFiles.jl](https://docs.julialang.org/en/v1/stdlib/DelimitedFiles/).
```

Comment:
DelimitedFiles is going to be removed from the stdlib soon, so probably better anticipate the wording and say "used to be in the stdlib": https://github.com/JuliaLang/julia/pull/45540.

Reply:
Maybe this should wait until it can mention a version number? Can't be before 1.9, and since 1.8 is not yet released.

Reply:
```suggestion
* The standard library contains [DelimitedFiles.jl](https://docs.julialang.org/en/v1/stdlib/DelimitedFiles/), at least until Julia 1.8.
```